Source/UI/NewUI/CommonUI/View/Component/SelectionButtonView.swift

```diff
@@ -6,7 +6,7 @@ protocol SelectionButtonViewDelegate: AnyObject {
 
 class SelectionButtonView: UIView {
     weak var delegate: SelectionButtonViewDelegate?
-    private var style: CellButtonStyle?
+    private(set) var style: CellButtonStyle?
```

Comment:
This was already being tested successfully previously without while being private, or was something being missed?

Reply:
it's needed for test case